app/src/app/api/query/write/__tests__/route.test.ts

```
@@ -0,0 +1,179 @@
+
+function makeRequest(body: unknown) {
+  return { json: async () => body } as Request;
+}
+
+/** Chainable drizzle query builder stub that resolves to `rows`. */
+function drizzleSelectChain(rows: unknown[]) {
+  const chain = {
+    from: () => chain,
+    where: () => chain,
+    limit: () => Promise.resolve(rows),
+    then: (resolve: (v: unknown[]) => unknown) => Promise.resolve(rows).then(resolve),
```

Biome (2.4.4)

[error] Do not add then to an object.

(lint/suspicious/noThenProperty)